src/frameName.h

```diff
@@ -75,6 +75,22 @@ class FrameName {
     Mutex& _thread_names_lock;
     ThreadMap& _thread_names;
     locale_t _saved_locale;
+    bool _includemm;
+    // Based on: https://docs.oracle.com/javase/specs/jvms/se7/html/jvms-4.html#:~:text=Table%C2%A04.5.%C2%A0Method%20access%20and%20property%20flags
+    const std::vector<std::pair<int, std::string>> access_flags = {
```

Comment:
As suggested here https://github.com/Granulate/gprofiler/pull/712#discussion_r1129777712 let's use a different construct so this builds without the std=11 flag.